src/charts.js

```diff
@@ -23,7 +23,58 @@ export function WeeklyStepsVsGoal(weekData, goal) {
   });
 }
 
-export function stepProgressBar(stepData, goal) {
+export function weeklySleepQualityChart(weekData) {
```

Comment:
The weekData being sent in here should already be broken down into 7 days. So this slice should be unnecessary. Please make sure we are passing in the correct data. This appears to be the case for both sleep data charts.